master/buildbot/www/config.py

```diff
@@ -37,6 +38,24 @@ def reconfigResource(self, new_config):
     def render_GET(self, request):
         return self.asyncRenderHelper(request, self.renderIndex)
 
+    def getEnvironmentVersions(self):
+        import sys
+        import twisted 
+        from buildbot import version as bbversion
+
+        pyversion = '.'.join(map(str, sys.version_info[:3]))
+
+        tx_version_info= (twisted.version.major,
+                          twisted.version.minor,
+                          twisted.version.micro)
+        txversion = '.'.join(map(str, tx_version_info))
+
+        return [
+            ('Python version', pyversion),
```

Comment:
indeed, you can say just python, and add ' version' in the UI.
This will allow i18n in the future, as version is a translatable string.